{
  "submission_id": "S91",
  "user_id": "U1",
  "problem_id": "P11",
  "timestamp": "2025-10-09T05:36:59Z",
  "status": "Running... (Testcase 7)",
  "language": "python",
  "test_results": [
    {
      "test_case_number": 1,
      "status": "runtime_error",
      "message": "Runtime Error: File \"/sandbox/temp/solution.py\", line 8\n    print(gcd(a, b))\nIndentationError: unexpected indent\nCommand exited with non-zero status 1",
      "execution_time": 0.01,
      "memory_usage": 7.99609375,
      "actual_output": "",
      "expected_output": "2",
      "input": "5698 4838\n"
    },
    {
      "test_case_number": 2,
      "status": "runtime_error",
      "message": "Runtime Error: File \"/sandbox/temp/solution.py\", line 8\n    print(gcd(a, b))\nIndentationError: unexpected indent\nCommand exited with non-zero status 1",
      "execution_time": 0.05,
      "memory_usage": 7.90234375,
      "actual_output": "",
      "expected_output": "3",
      "input": "46926 42591\n"
    },
    {
      "test_case_number": 3,
      "status": "runtime_error",
      "message": "Runtime Error: File \"/sandbox/temp/solution.py\", line 8\n    print(gcd(a, b))\nIndentationError: unexpected indent\nCommand exited with non-zero status 1",
      "execution_time": 0.01,
      "memory_usage": 7.7734375,
      "actual_output": "",
      "expected_output": "3",
      "input": "1953 192\n"
    },
    {
      "test_case_number": 4,
      "status": "runtime_error",
      "message": "Runtime Error: File \"/sandbox/temp/solution.py\", line 8\n    print(gcd(a, b))\nIndentationError: unexpected indent\nCommand exited with non-zero status 1",
      "execution_time": 0.03,
      "memory_usage": 7.5859375,
      "actual_output": "",
      "expected_output": "1",
      "input": "41906 7297\n"
    },
    {
      "test_case_number": 5,
      "status": "runtime_error",
      "message": "Runtime Error: File \"/sandbox/temp/solution.py\", line 8\n    print(gcd(a, b))\nIndentationError: unexpected indent\nCommand exited with non-zero status 1",
      "execution_time": 0.0,
      "memory_usage": 7.796875,
      "actual_output": "",
      "expected_output": "1",
      "input": "1 1\n"
    },
    {
      "test_case_number": 6,
      "status": "runtime_error",
      "message": "Runtime Error: File \"/sandbox/temp/solution.py\", line 8\n    print(gcd(a, b))\nIndentationError: unexpected indent\nCommand exited with non-zero status 1",
      "execution_time": 0.01,
      "memory_usage": 7.91796875,
      "actual_output": "",
      "expected_output": "2",
      "input": "48266 6718\n"
    },
    {
      "test_case_number": 7,
      "status": "runtime_error",
      "message": "Runtime Error: File \"/sandbox/temp/solution.py\", line 8\n    print(gcd(a, b))\nIndentationError: unexpected indent\nCommand exited with non-zero status 1",
      "execution_time": 0.01,
      "memory_usage": 7.765625,
      "actual_output": "",
      "expected_output": "2",
      "input": "36782 13032\n"
    }
  ]
}